### PR TITLE
Capstone group project hit api

### PR DESCRIPTION
**Pull request by osbentech**
This is our group Javascript capstone project.
below are the features we implemented:
- Find external API
- Choose assets
- Set up the project with Webpack and Jest
- Add header
- Add footer
- Display list of items on the Homepage
- Display Comments pop up with selected item's details
- Display number of likes for each item on the Homepage
- Display comments for a given item on the Comments pop-up
- Create feature: add new like
- Create feature: add new comment
- Add all items counter on the Homepage
- Add comments counter
- Add tests for items counter
- Add tests for comments counter

### CODE REVIEW

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Team,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
- [ ] Nice working on your video presentation the maximum length of your video should be 5 minutes and your video needs to have a highlight about a piece of the code you find interesting or are proud of making.

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,154 @@
...
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#authors">Authors</a></li>
+    <li><a href="#support">Support</a></li>
+    <li><a href="#license">License</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+# JavaScript Capstone Project
+
```
Comment (andersonlebon) [inline README.md:32]: - [ **OPTIONAL** ] What do you think if you give your project a meaningful name with a short idea about its content to make it more professional? e.g.: iCinema, MoviesInfos,... a thing similar to that

**Review by andersonlebon — APPROVED** [review]
Hi @**Team** :wave: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by osbentech — COMMENTED** [review]
Great effort Team,

Your project is complete, amazing job!  🥇🥇 
There is nothing else to say other than it's time to marge it.
Congratulations!

**Pull request merged**